GateWatcher/aioniq/_meta/smart-descriptions.json

```
@@ -0,0 +1,91 @@
+[{
+    "value": "Infected payload {file.name} hosted on {url.domain} was blocked: {gatewatcher.malcore.detail_threat_found}",
+    "conditions": [
+        {
+            "field": "gatewatcher.type",
+            "value": "malcore"
+        },
+        {
+            "field": "state",
```

```suggestion
            "field": "gatewatcher.state",
```

I am not sure that only `"state"` will work

---

fixed